addons/mail/static/src/models/composer_view.js

```diff
@@ -227,9 +227,7 @@ registerModel({
          * @param {MouseEvent} ev
          */
         onClickEmoji(ev) {
-            if (this.textInputComponent) {
-                this.textInputComponent.saveStateInStore();
```

Comment:
Could remove this field/useComponentToModel, as it was its only purpose? Unless other PR will need it again?

Reply:
Pretty sure it's not useful anymore, so I can just remove it in this PR too. Just made the changes.